setup.py

```diff
@@ -0,0 +1,9 @@
+import setuptools
+
+setuptools.setup(
+    name="data-pipeline",
+    version="0.0.1",
+    packages=setuptools.find_packages(),
+    install_requires=["apache-beam[gcp]==2.38.0"],
```

Comment:
Maybe a silly question. The version is specified in requirements.txt, why do we need to specify it in both places?

Reply:
Packages listed in `setup.py` are used for packaging distributions only (subset), whereas `requirements.txt` contains additional dependencies used during development and testing (superset).

https://packaging.python.org/en/latest/discussions/install-requires-vs-requirements/#requirements-files

This is related to using the `--setup_file` pipeline option, since it is required when the package is split across multiple files.

https://beam.apache.org/documentation/sdks/python-pipeline-dependencies/#multiple-file-dependencies